sqlmesh/core/loader.py

```diff
@@ -42,6 +45,24 @@
 logger = logging.getLogger(__name__)
 
 
+_cache = None
```

Comment:
Why do you need this global state? The cache instance should be very lightweight and it should be ok to just create an instance when loading.

Reply:
every time?, this makes it so that every process initializes the cache once

Reply:
What is there to initialize?

Reply:
in order for each process to have access to the state, the pattern i've found online is to use a global